csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html

```diff
@@ -0,0 +1,186 @@
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="{% static 'tech_fair_css.css' %}"/>
+	<title>SFU Tech Fair 2022</title>
+
+
+  <script>
+    // Resize the frame bounds to show all its contents on the current page without scrolling.
+    window.addEventListener('load', () => {
+        const frame = document.getElementById("schedule-frame");
+        frame.style.height = `${frame.contentWindow.document.body.offsetHeight}px`;
+    })
+  </script>
+</head>
+
+<body>
+
+ 
+ <div class="topnav">
+  <img class="logo" src="{% static 'csss_logo.png' %}" alt="Computing Science Student Soicety Logo">
```

Comment:
1. you have a typo
2. These are supposed to link back to the main page

Reply:
1. Resolved.
2. Resolved.